Site/SiteAnalyticsModule.php

```diff
@@ -335,6 +496,39 @@ protected function getGoogleAnalyticsCommand($command)
 	}
 
 	// }}}
+	// {{{ protected function getFacebookPixelCommand()
+
+	protected function getFacebookPixelCommand($command)
+	{
+		$method  = null;
+		$options = null;
+
+		if (is_array($command)) {
+			$method = array_shift($command);
+
+			foreach ($command as $part) {
+				$quoted_part = (is_float($part) || is_int($part))
+					? $part
+					: SwatString::quoteJavaScriptString($part);
+
+				$options.= sprintf(
+					', %s',
+					$quoted_part
+				);
+			}
+		} else {
+			$method = $command;
+		}
+
+		return sprintf(
+			'fbq(%s, %s%s);',
+			SwatString::quoteJavaScriptString('track'),
```

Comment:
since this is a static string you could put it in the template pre-quoted rather than sprintfing it.